src/workspaces/container/WorkspaceContainer.ts

```diff
@@ -244,60 +229,75 @@ export const wrapWorkspace = (opts: WrapWorkspaceOptions): WrapWorkspaceFn => {
         useCloudEnvironmentPolling(name, namespace, workspace);
       const { apps, refreshApps, lastRefresh } = useAppPolling(name, namespace, workspace);
 
-      if (accessError) {
-        return h(FooterWrapper, [h(TopBar), h(WorkspaceAccessError)]);
-      }
+      return h(FooterWrapper, [
+        h(TopBar, { title: 'Workspaces', href: Nav.getLink('workspaces') }, [
+          div({ style: Style.breadcrumb.breadcrumb }, [
+            div({ style: Style.noWrapEllipsis }, breadcrumbs(props)),
+            h2({ style: Style.breadcrumb.textUnderBreadcrumb }, [
+              (_.isFunction(title) ? title(props) : title) || `${namespace}/${name}`,
+            ]),
+          ]),
+        ]),
+        (() => {
```

Comment:
I'm having trouble understanding… why does this have to be a function?

Reply:
It doesn't _have_ to be. This uses an immediately invoked function expression to conditionally render the children of `FooterWrapper`.

There are a couple of possible alternatives...
```ts
let content
if (...) {
  content = ...
} else {
  content = ...
}

return h(FooterWrapper, [..., content])
```

```ts
return h(FooterWrapper, [
  ...
  cond(
    [..., () => ...]
    [..., () => ...]
  )
])
```

Reply:
That was the conclusion I came to… the function seems unnecessarily complex to me, but I don't feel strongly about it.